src/SDKs/Azure.Core/data-plane/Azure.Core.Tests/SocketTransport/SocketClientTransport.cs

```diff
@@ -0,0 +1,265 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using Azure.Core.Buffers;
+using Azure.Core.Http.Pipeline;
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Security;
+using System.Net.Sockets;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using static System.Buffers.Text.Encodings;
+
+namespace Azure.Core.Http
+{
+    public class SocketClientTransport : PipelineTransport
```

Comment:
Yeah, this is what I was talking about. It should/must be moved back to labs. I will do the work if you assign it to me.

Reply:
Filed https://github.com/Azure/azure-sdk-for-net/issues/5219 for this.